src/frontend/components/SøknadsSteg/Utvidet-DinLivssituasjon/types-and-utilities.ts

```diff
@@ -0,0 +1,19 @@
+export enum Årsak {
+    SEPARERT = 'omdeg.velgårsak.separert',
+    SKILT = 'omdeg.velgårsak.skilt',
+    BRUDD_SAMBOER = 'omdeg.velgårsak.bruddsamboer',
+    BODD_ALENE = 'omdeg.velgårsak.boddalene',
+    ENKE_ENKEMANN = 'omdeg.velgårsak.enkeenkemann',
+    FENGSEL_VARETEKT = 'omdeg.velgårsak.fengselvaretekt',
+    BURDD_GIFT = 'omdeg.velgårsak.burddgift',
```

Comment:
BURDD?

Også, kommer disse fra et kodeverk eller er det bare noe vi bestemmer?

Reply:
Det stod BURDD (istede for brudd) i språkfila på googledrive. Har nå endret over hele rekka til "brudd"

De id'ene i enumen brukte jeg bare fordi det virka kjekt å få riktig språknøkkel på den måten. Men jeg skal prøve å endre litt i tråd med det Daphne nevner under